document/CoreFormat.mdk

```diff
@@ -1891,6 +1989,23 @@ a stable, opaque identifier for the rule.
 `"CA2101"`
 ~
 
+~ IsoNote
+Rule identifiers must be stable for two reasons:
+
+1. So specific checks can be disabled in build automation scripts, without the risks
```

Comment:
"So build automation scripts can refer to specific checks, for example, to disable them, without the risk of a script breaking if a rule id changes."